.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script installs the built kata-containers in the test cluster,
+# and configure a runtime.
+
+scripts_dir=$(dirname $0)
+deployments_dir=${scripts_dir}/deployments
+configs_dir=${scripts_dir}/configs
+
+source ${scripts_dir}/../../lib.sh
+
+oc project default
+
+worker_nodes=$(oc get nodes |  awk '{if ($3 == "worker") { print $1 } }')
+num_nodes=$(echo $worker_nodes | wc -w)
+[ $num_nodes -ne 0 ] || die "No worker nodes detected. Something is wrong with the cluster"
+
+info "Set installing state annotation and label on all worker nodes"
+for node in $worker_nodes; do
+    info "Deploy kata in worker node $node"
+    oc annotate --overwrite nodes $node \
+	    kata-install-daemon.v1.openshift.com/state=installing
+    oc label --overwrite node $node kata-deploy=true
+done
+
+info "Applying the kata installer daemonset"
+if [ -z "$KATA_INSTALLER_IMG" ]; then
+	# The yaml file uses $IMAGE_FORMAT which gives the
+	# registry/image_stream:image format
+	export KATA_INSTALLER_IMG=$(echo $IMAGE_FORMAT"kata-installer" \
+		| envsubst)
+fi
+envsubst < ${deployments_dir}/daemonset_kata-installer.yaml.in | oc apply -f -
+
+oc get pods
+oc get ds
+ds_pods=($(oc get pods | awk '{if ($1 ~ "kata-deploy") { print $1 } }'))
+cnt=5
+while [[ ${#ds_pods[@]} -gt 0 && $cnt -ne 0 ]]; do
+        sleep 120
```

Comment:
That's a long time; does it need to be this long?

Reply:
I ran this script in OpenShift CI many times, the kata-deploy daemonset never finished before 2min. Usually it takes 3 iterations to finish (~6 min). So I don't think 2 min on this sleep is a long time.